### PR TITLE
Data node selector doc update for 4.0

### LINKED ISSUE #1197: Document the 4.0 changes in the Data Node selector
**Description**

As of today, 4.0 Release Notes mention a few changes in the Data Node selector, with too few details.

**Expected change**

Changes are integrated in the Release notes, the doc page for the `data_node_selector` control is updated, including screen shots.

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Addresses #1197.

### CODE REVIEW

File: docs/refmans/gui/viselements/corelements/data_node_selector.md_template [hunk 1]
```diff
@@ -2,62 +2,243 @@ Displays a list of the Data Node entities that can be selected.
...
+[`Scope.SCENARIO`](../../../reference/pkg_taipy/pkg_common/pkg_config/Scope/index.md).
 
-When the user selects a data node, the [*on_change*](#p-on_change) callback is invoked so that
-the application can use the selected value. The value is set to the [*value*](#p-value) property.
+When the user selects a data node, the [*on_change*](#p-on_change) callback is triggered allowing
+the application to respond to the user's selection. The selected node's value is reflected in the
+control's [*value*](#p-value) property.
 
-<h2>Pins</h2/>
+At the top of the control, a toolbar provides additional functionality via a series of icons. These
```
Comment (FredLL-Avaiga) [inline docs/refmans/gui/viselements/corelements/data_node_selector.md_template:29]: series => serie ?

File: docs/refmans/gui/viselements/corelements/data_node_selector.md_template [hunk 2]
```diff
@@ -2,62 +2,243 @@ Displays a list of the Data Node entities that can be selected.
...
+When the user presses the
+<svg class="mui-icon" viewBox="0 0 24 24"><use href="../../mui-icons.svg#sort_by_alpha"/></svg>
+icon (Sort), the sorting configuration dialog appears:
+<figure class="tp-center">
+  <img src="../data_node_selector-sort2-d.png" class="visible-dark"  width="80%"/>
+  <img src="../data_node_selector-sort2-l.png" class="visible-light" width="80%"/>
+  <figcaption>Sorting configuration dialog</figcaption>
+</figure>
+
+The user can select a property from the dropdown and Specify the sorting order: 'asc' (Ascending) or
```
Comment (FredLL-Avaiga) [inline docs/refmans/gui/viselements/corelements/data_node_selector.md_template]: Specify => specify

**Review by FredLL-Avaiga — APPROVED** [review]

File: docs/refmans/gui/viselements/corelements/data_node_selector.md_template (hunk 2 above)
Comment (FabienLelaquais) [reply docs/refmans/gui/viselements/corelements/data_node_selector.md_template]: Good catch!

File: docs/refmans/gui/viselements/corelements/data_node_selector.md_template (hunk 1 above)
Comment (FabienLelaquais) [reply docs/refmans/gui/viselements/corelements/data_node_selector.md_template:29]: Nope. Series is always ' plural'. Thx!

**Pull request merged**